src/main/java/com/influxdb/v3/client/PointValues.java

```diff
@@ -435,6 +435,17 @@ public PointValues setField(@Nonnull final String field, @Nullable final Object
     return putField(field, value);
   }
 
+  /**
+   * Add a null field.
+   *
+   * @param field the field name
+   * @return this
+   */
+  @Nonnull
+  public PointValues setNullField(@Nonnull final String field) {
+    return putField(field, null);
+  }
+
```

Comment:
This is not necessary.

Reply:
If I use setField(), it will get a compiler error because setField only accepts the primitive value
Should I just leave this case and do nothing if the cast is wrong, but if do that the field name will not be stored in the field map

Reply:
`setField(@Nonnull final String field, @Nullable final Object value)` accepts `Object`

Reply:
But I thought if the typecast fails, we will set the field to null

Reply:
Sorry for the misunderstanding; it was my fault. If we are not able to cast to the type defined by the metadata, we will use setField for mapping to PointValues. For mapping to the ‘raw’ object, we will return the original value.

Reply:
No problem Jakub
I will return the original value when cast fails now